src/AvBaseInput.js

```diff
@@ -149,6 +157,9 @@ export default class AvBaseInput extends Component {
     if (this.props.type === 'checkbox') {
       return event.target.checked ? this.props.trueValue : this.props.falseValue;
     }
+    if (this.props.type === 'select' && this.props.multiple){
+      return [...event.target.options].filter(({selected}) => selected).map(({value}) => value);
```

Comment:
I did test it as event.target.options.filter(({selected}) => selected).map(({value}) => value); and I get a strange resetof selected values in UI when I change another select. It works in test but not in real app. Didn't dig any deeper as to why this happens. So I'll leave it as it is for now.